src/pages/NewChatPage.js

```diff
@@ -77,6 +79,13 @@ class NewChatPage extends Component {
     getSections() {
         const sections = [];
 
+        sections.push({
+            title: this.props.translate('iou.recents'),
+            data: this.state.recentReports,
+            shouldShow: this.state.recentReports.length > 0,
```

Comment:
`!_.isEmpty` would be preferred here ([example](https://github.com/Expensify/Expensify.cash/blob/55a1c66e7670b4b0f39de8485c3dcd916d6dc4da/src/pages/SettingsPage.js#L55))

Reply:
I just changed the code.

And I noticed that [OptionsSelector](https://github.com/Expensify/Expensify.cash/blob/main/src/components/OptionsSelector.js) ignores shouldShow condition but it requires it as prop.